src/analyze/analyze.c

```diff
@@ -1832,6 +2010,50 @@ static int do_security(int argc, char *argv[], void *userdata) {
         return analyze_security(bus, strv_skip(argv, 1), 0);
 }
 
+static int parse_arg_color_style(DotArgColorStyle color_or_style, const char *arg_dot_color_style) {
+        char delim[] = ":";
```

Comment:
why put this in an array? why not specify that literally when invoking extract_first_word()? (also, why isn't this static const?)

Reply:
changed